src/components/Account/ReputationView.tsx

```diff
@@ -0,0 +1,22 @@
+import Tooltip from "rc-tooltip";
+import * as React from "react";
+
+interface IProps {
+  daoName?: string;
+  reputation: number;
+  totalReputation: number;
+}
+
+export default class ReputationView extends React.Component<IProps, null> {
+  public render() {
+    const { daoName, reputation, totalReputation } = this.props;
+
+    return (
+      <Tooltip overlay={<span>{reputation} {daoName || ''} Reputation in total</span>}>
```

Comment:
the reputation count here should be fixed/rounded to two decimals

Reply:
also should probably use toLocaleString() to look nicer

Reply:
https://github.com/daostack/alchemy/pull/71/commits/b0a1d80240a5c5b13a4348d00c396741d535a889